docs/templates/partner-plugin-template/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+nav_title: # Set a title for the navigation menu
```

Comment:
Same comment here about adding an example title for reference

Reply:
This one shouldn't even be here, I forgot to remove.